public/app/features/library-panels/components/LibraryPanelsSearch/LibraryPanelsSearch.tsx

```diff
@@ -48,9 +48,11 @@ export const LibraryPanelsSearch = ({
   useDebounce(() => setDebouncedSearchQuery(searchQuery), 200, [searchQuery]);
 
   const [sortDirection, setSortDirection] = useState<SelectableValue<string>>({});
-  const [folderFilter, setFolderFilter] = useState<string[]>(currentFolderUID ? [currentFolderUID] : []);
   const [panelFilter, setPanelFilter] = useState<string[]>([]);
 
+  // Existing code expects an array, so just match that for now
```

Comment:
Not sure I understand this comment - is this a leftover? Seems like you have updated the code to respect `string` rather than `[]`.

Reply:
yup thanks, good catch.